src/favorites/favorites.controller.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Post,
+} from '@nestjs/common';
+import {
+  ApiBadRequestResponse,
+  ApiCreatedResponse,
+  ApiNoContentResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+  ApiUnprocessableEntityResponse,
+} from '@nestjs/swagger';
+import { UUIDParam } from 'src/common/helpers/request.decorators';
+import { Favorites } from './favorites.model';
+import { FavoritesService } from './favorites.service';
+import { IFavoritesResponse } from './interfaces/favorites.interface';
+
+@ApiTags('Favorites')
+@Controller('favs')
+export class FavoritesController {
+  constructor(private favoritesService: FavoritesService) {}
+
+  @ApiOperation({ summary: 'Get all favorites' })
+  @ApiResponse({ status: HttpStatus.OK, type: [Favorites] })
```

Comment:
Response example in swagger should show whole instances but currently it shows only instance ids.